libs/digger_config/digger_config.go

```
@@ -30,6 +30,17 @@ type FileSystemTerragruntDirWalker struct {
 type FileSystemModuleDirWalker struct {
 }
 
+func ReadDiggerYmlFileContents(dir string) (string, error) {
+	var diggerYmlBytes []byte
+	diggerYmlBytes, err := os.ReadFile(path.Join(dir, "digger.yml"))
+	if err != nil {
+		// if file doesn't exist look for digger.yaml instead
+		diggerYmlBytes, err = os.ReadFile(path.Join(dir, "digger.yaml"))
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)